server/repository/src/migrations/v1_02_00/invoice.rs

```diff
@@ -7,13 +7,23 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
         connection,
         r#"
             ALTER TYPE invoice_type ADD VALUE 'PRESCRIPTION';
+            ALTER TABLE invoice ADD clinician_id TEXT REFERENCES clinician(id);
```

Comment:
was hoping we reduce duplication with manual migrations: https://github.com/openmsupply/open-msupply/compare/1951-Endpoint-Update-Prescription...1951-Endpoint-Update-Prescription(Reduce-duplication-in-migrations)